lib/fluent/plugin/in_wmi.rb

```diff
@@ -7,25 +7,34 @@
 module Fluent::Plugin
   class WmiInput < Fluent::Plugin::Input
     Fluent::Plugin.register_input("wmi", self)
+    helpers :timer
 
     desc "The value is the tag assigned to the generated events."
     config_param :tag, :string
-
     desc "The value is the instance name of WMI."
     config_param :class_name, :string
+    desc "Determine the rate to emit metrics as events."
+    config_param :interval, :time, default: 10
 
     def configure(conf)
       super
-
       raise Fluent::ConfigError, "class_name is required." unless @class_name
+      begin
+        WmiLite::Wmi.new.instances_of(@class_name)
+      rescue WmiLite::WmiException
+        raise Fluent::ConfigError, "#{@class_name} is invalid class_name."
+      end
+      raise Fluent::ConfigError, "interval must be large than 1." if @interval < 1
```

Comment:
nits: Missing *r* in comparative form for **large**.
```suggestion
      raise Fluent::ConfigError, "interval must be larger than 1." if @interval < 1
```